src/main/typescript/reducers/authenticationReducer.ts

```diff
@@ -20,7 +20,7 @@ import { AuthenticationConstants } from "../constants/authenticationConstants"
 export const authenticationReducer: Reducer<Authentication> = (state = empty, action) => {
     switch (action.type) {
         case AuthenticationConstants.AUTH_LOGIN_FULFILLED: {
-            return { ...state, isAuthenticated: true, isAuthenticating:false }
+            return { ...state, isAuthenticated: true, isAuthenticating:false, authenticationError: "" }
```

Comment:
this `authenticationError` reset (and below) is supposed to go in the `PENDING` case. Also, not `authenticationError: ""` but `authenticationError: undefined`, I think. (Maybe not though, not sure if `"" === undefined`; JS sometimes has weird rules...)